cmd/image/qcow2ova/password-generator.go

```diff
@@ -0,0 +1,16 @@
+package qcow2ova
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+)
+
+// Generates the password of length n
+func GeneratePassword(n int) (string, error) {
```

Comment:
Since we are returning base64 encoded string, it's better to rename the return var to something like b64Password or b64String